draft-ietf-acme-acme.md

```diff
@@ -51,16 +51,23 @@ normative:
       name: Juraj Somorovsky
       ins: J. Somorovsky
     target: https://dl.acm.org/citation.cfm?id=2813657
+  REST:
+    title: Architectural Styles and the Design of Network-based Software Architectures
+    author:
+      name: Roy Thomas Fielding
+      ins: R. Fielding
+    date: 2000
+    target: http://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm
 
 
 --- abstract
 
-Certificates in PKI using X.509 (PKIX) are used for a number of purposes,
+Public Key Infrastructure X.509 (PKIX) certificates are used for a number of purposes,
 the most significant of which is the authentication of domain names.  Thus,
-certificate authorities in the Web PKI are trusted to verify that an applicant
```

Comment:
I'm a :-1: on changing all instances of "Certificate authority" to "Certification Authority". The later is not the predominant definition that folks are familiar with.

Reply:
I'm certainly not planning on dying on this hill, but there there is some precedent to using it based on RFC 2459, 328, and 5280 as well as slew of other RFCs.  And, RFC 4949 says "certificate authority" SHOULD NOT be used in Standards Track documents.

Reply:
Ok. I'll cave on this but I think its unfortunate that we're avoiding a term with common industry wide usage to favour one that is seemingly only favoured by folks already steeped in the nitty gritty of these standards who likely need the least help understanding an RFC and its contents.

Reply:
I note that the CABF Baseline Requirements also use Certification Authority.  I agree that it's unpleasant, but it seems to be proper.